src/System.Security.Cryptography.Encoding/tests/System.Security.Cryptography.Encoding.Tests.csproj

```diff
@@ -17,11 +17,22 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AsnEncodedData.cs" />
+    <Compile Include="DerEncoderTests.cs" />
+    <Compile Include="DerSequenceReaderTests.cs" />
     <Compile Include="Oid.cs" />
     <Compile Include="OidCollection.cs" />
+    <Compile Include="$(CommonPath)\System\Security\Cryptography\DerEncoder.cs">
+      <Link>Common\System\Security\Cryptography\DerEncoder.cs</Link>
+    </Compile>
```

Comment:
Ish :) This is a class (well, two) instead of the whole library; and I actually want to make what they're doing public and just use the public surface area.  Hopefully this is a point-in-time solution :).